app/assets/stylesheets/common/admin/customize.scss

```diff
@@ -923,3 +923,86 @@ table.permalinks {
     }
   }
 }
+
+.form-templates {
+  &--info {
+    margin-top: 1rem;
+  }
+
+  &--table {
+    margin-bottom: 1rem;
+
+    .admin-list-item .action {
+      text-align: right;
+    }
+  }
+
+  &--form {
+    input {
+      width: 300px;
+    }
+
+    .ace-wrapper {
+      position: relative;
+      height: calc(100vh - 450px);
+      min-height: 200px;
+      width: 100%;
+      box-shadow: shadow("footer-nav");
+      border-radius: 4px;
+    }
+
+    .ace_editor {
+      border-radius: 4px;
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      bottom: 0;
+    }
+
+    .ace_placeholder {
+      font-family: inherit;
+      font-size: var(--font-up-1);
+      color: var(--primary-high);
+    }
+
+    .footer-buttons {
+      display: flex;
+      gap: 0.5rem;
+      .btn-danger {
+        margin-left: auto;
+      }
+    }
+  }
+
+  &--quick-insert-field-buttons {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    margin-left: 1rem;
+
+    span {
+      margin-right: 0.25rem;
+    }
+
+    .btn {
```

Comment:
Nitpicky, but is the button styling here necessary? Shouldn't regular buttons work just fine?

Reply:
Ah yes, I don't think I need this portion
```
    &:hover,
      &:focus {
        background-color: var(--primary-low);
        color: var(--primary);
      }
```

but I think in the mockup we had borders to the right.

I'll update the css